### PR TITLE
Fix Issue 362: User configurable protoc location

### PR DESCRIPTION
**Pull request by nmost**

### CODE REVIEW

File: scalapbc/src/main/scala/com/trueaccord/scalapb/ScalaPBC.scala [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
   version: String = "-v310",
   throwException: Boolean = false,
-  args: Seq[String] = Seq.empty)
+  args: Seq[String] = Seq.empty,
+  customProtocLocation: Option[String] = None)
 
 class ScalaPbcException(msg: String) extends RuntimeException(msg)
 
 object ScalaPBC {
+  private val customPathArgument =  "--protocPath="
```
Comment (thesamet) [inline scalapbc/src/main/scala/com/trueaccord/scalapb/ScalaPBC.scala]: Let's make it `--protoc=` (without the `Path`)

Comment (nmost) [reply scalapbc/src/main/scala/com/trueaccord/scalapb/ScalaPBC.scala]: done

**Comment by nmost** [comment]
What's the process like to get this released? I'm struggling mightily to pull this code down and get it building within our nix environment and being able to just pull the jar off of maven would help me out greatly

**Comment by thesamet** [comment]
I'll get it released this week.

**Comment by thesamet** [comment]
Released 0.6.7

**Pull request merged**